utils.py

```diff
@@ -0,0 +1,20 @@
+import subprocess
+
+
+def pip_install(dependancy):
+    try:
+        subprocess.run(
+            f"pip install {dependancy}", check=True, stdout=subprocess.PIPE, text=True
```

Comment:
```suggestion
            f"pip install --user {dependancy}", check=True, stdout=subprocess.PIPE, text=True
```

I'm assuming for now at least, we'll be running these commands without VirtualEnv so lets not pollute the system packages at least.